docs/EN_US/HPCCSystemAdmin/SA-Mods/DAFtslave.xml

```diff
@@ -14,17 +14,19 @@
     <para>The dafilesrv is a daemon that runs on system servers or physical
     locations on the HPCC system. This daemon enables the HPCC components to
     have file access to physical file locations. File access could be
-    spraying, despraying, copying, etc. It is also used to enable file
-    security.</para>
-
-    <para>The dafilesrv daemon runs as a process on every Thor or Roxie node.
-    The daemon will typically keep running even if your system is stopped.
-    That is an important fact to keep in mind if you are stopping your system
-    for maintenance. Particularly for installations or upgrades. Even if you
-    issue the HPCC system stop command, you would still need to stop the
-    dafilesrv daemon. Issue the following command to stop the dafilesrv
-    daemon. Then verify that dafilesrv is not running before running the
-    installation package. </para>
+    spraying, despraying, copying, accessing logs, etc. Anything that requires
+    a file from another node, must have dafilesrv running on that node. The
+    dafilesrv daemon can be configured to allow SSL connections only, which
+    secures remote access to physical files.</para>
+
+    <para>The dafilesrv daemon runs as a process on every HPCC node. The
+    daemon will typically keep running even if your system is stopped. That is
+    an important fact to keep in mind if you are stopping your system for
+    maintenance. Particularly for installations or upgrades. Even if you issue
+    the HPCC system stop command, you would still need to stop the dafilesrv
+    daemon. Issue the following command to stop the dafilesrv daemon. Then
+    verify that dafilesrv is not running before running the installation
+    package.</para>
```

Comment:
1) Doesn't the installer handle this automatically? 
2) The two sentences are better as a single sentence : Issue the following command to stop the dafilesrv daemon, then verify that dafilesrv is not running before running the installation package. 
3) Doesn't say HOW to verify that it is not running.